pkg/authorization/bootstrap/policy.go

```diff
@@ -24,13 +24,18 @@ import (
 
 const (
 	SystemKcpClusterWorkspaceAccessGroup = "system:kcp:clusterworkspace:access"
-	SystemKcpClusterWorkspaceAdminGroup  = "system:kcp:clusterworkspace:admin"
+	// An admin group per cluster workspace.
+	// Members of this group have all permissions in the referenced cluster workspace (capped by maximal permission policy).
+	SystemKcpClusterWorkspaceAdminGroup = "system:kcp:clusterworkspace:admin"
+	// A global admin group.
+	// Members of this group have all permissions across all cluster workspaces.
+	SystemKcpAdminGroup = "system:kcp:admin"
```

Comment:
@csams fyi, this is the new group I was mentioning. Little less than system:masters, i.e. it goes through authz.